src/app/app.component.scss

```diff
@@ -24,6 +24,8 @@ ion-menu.md ion-list-header {
   padding-right: 18px;
   padding-left: 18px;
 
+  font-size: min(1.375rem, 36px);
```

Comment:
This causes visual changes on MD:

| `main` | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-conference-app/assets/2721089/d21b60b1-fb41-4550-871c-d7a5504ad9dc) | ![image](https://github.com/ionic-team/ionic-conference-app/assets/2721089/53ca528f-95f2-49c7-8265-c7ebcde7edc7) |


Changes look good on iOS though